code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -96,8 +96,8 @@
 	foldable = null //special form fitted boxes should not be foldable.
 
 /obj/item/storage/box/survival
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "site survival kit"
+	desc = "A survival box containing a variety of supplies necessary to ensure survival. This one possesses oxygen, as shown by the red panel and two striped yellow lines."
```

Comment:
Get rid of the "survival" as well as the anecdote about the red panel and yellow stripes. Use "contains" instead of "possesses". Also drop the flowery writing.

Reply:
it's the most common object in the game, it deserves a detailed description as to why it was in your bag every shift.

Reply:
changed.